src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,112 @@
+﻿import React, { Suspense, useEffect, useRef, useState } from 'react'
+import { ColorResult } from '@uiw/react-color'
+import styles from '@/components/ColorPicker/ColorPicker.module.scss'
+import { Tooltip } from '@/components'
+
+type ColorType = {
+  hsla: string;
+  rgba: string;
+  hex: string
+}
+type ColorPickerProps = {
+  value?: ColorType,
+  onChange?: (color: ColorType) => void;
+  description?: string;
+  lazyLoadingText?: string;
+}
+
+const ColorPicker: React.FC<ColorPickerProps> = (props) => {
+  const [Sketch, setSketch] = useState<any>(null)
+  const containerRef = useRef<HTMLDivElement>(null)
+  const { 
+    onChange = () => {},
+    lazyLoadingText = 'Loading...' 
+  } = props
+  const colorValue = props.value
+    ? props.value
+    : {
+        hex: '#ffffff',
+        hsla: 'hsla(0, 0%, 100%, 1)',
+        rgba: 'rgba(255, 255, 255, 1)',
+      }
```

Comment:
You can use object destruction with default values here instead of the ternary operator. Looks a bit cleaner.
```suggestion
      const {
        value = {
          hex: '#ffffff',
          hsla: 'hsla(0, 0%, 100%, 1)',
          rgba: 'rgba(255, 255, 255, 1)',
        },
      } = props
```